zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -133,9 +132,7 @@ private void initialiseIndices() {
       return;
     }
 
-    final var existingIndexNames =
-        Collections.synchronizedSet(
-            searchEngineClient.getMappings(allIndexNames(), MappingSource.INDEX).keySet());
+    final var existingIndexNames = Collections.synchronizedSet(existingIndexNames());
```

Comment:
🔧 I think we actually can removed this now. It is only used for logging. Feels a bit like a waste.